BlazarService/src/main/java/com/hubspot/blazar/github/BackfillGitHubDataCommand.java

```diff
@@ -13,26 +20,16 @@
 import com.hubspot.horizon.HttpRequest;
 import com.hubspot.horizon.HttpRequest.Method;
 import com.hubspot.horizon.ning.NingHttpClient;
+
 import io.dropwizard.cli.ConfiguredCommand;
 import io.dropwizard.jackson.Jackson;
 import io.dropwizard.setup.Bootstrap;
 import net.sourceforge.argparse4j.inf.Namespace;
 import net.sourceforge.argparse4j.inf.Subparser;
-import org.kohsuke.github.GHOrganization;
-import org.kohsuke.github.GHRepository;
-import org.kohsuke.github.GitHub;
-
-import java.io.IOException;
-import java.util.Map.Entry;
 
 public class BackfillGitHubDataCommand extends ConfiguredCommand<BlazarConfiguration> {
-  private final HttpClient httpClient;
-
   public BackfillGitHubDataCommand() {
     super("backfill", "Reprocess all repo and branch data");
-
-    ObjectMapper mapper = Jackson.newObjectMapper().setSerializationInclusion(Include.NON_NULL);
-    this.httpClient = new NingHttpClient(HttpConfig.newBuilder().setObjectMapper(mapper).build());
```

Comment:
FYI instantiating the HttpClient in the constructor was keeping the process from exiting when a DB migration was run (nothing was closing the connection pool :disappointed: )